tests/vm_test.go

```diff
@@ -1921,6 +1923,72 @@ status:
 			}, 2*time.Minute, 1*time.Second).Should(Succeed())
 		})
 	})
+
+	Context("[Serial] when node becomes unhealthy", Serial, func() {
+		const componentName = "virt-handler"
+		var nodeName string
+		var numberOfVirtHandler int32
```

Comment:
Might be better to initialize it here. Otherwhise it could be undefined if the test case fails before initializing it. (possibly adds flakiness)

Reply:
`numberOfVirtHandler` has been dropped. And regarding `nodeName` I can't know the value before the VMI is scheduled. I can change the VM creation to a `BeforeEach` if that's helpful.